.github/workflows/lint.yml

```diff
@@ -96,6 +96,7 @@ jobs:
                      --known-failure app/app-platform/ContentApp.h \
                      --known-failure app/app-platform/ContentAppPlatform.cpp \
                      --known-failure app/app-platform/ContentAppPlatform.h \
+                     --known-failure controller/jcm/TrustVerification.h \
```

Comment:
We are not allowed to add new files without build rule verification.

Adding to this list means either an oversight (we should not do that) or that we intentionally cannot add these to build rules because our includes violate layering/dependency rules that GN checks (we should not do that either)